api/log_test.go

```diff
@@ -52,11 +52,11 @@ func (s *S) TestAddLogsHandler(c *check.C) {
 	c.Assert(err, check.IsNil)
 	baseTime = baseTime.Local()
 	bodyStr := `
-	{"date": "2015-06-16T15:00:00.000Z", "message": "msg1", "source": "web", "appname": "myapp1", "unit": "unit1"}
```

Comment:
this change may break the tsuru client ? @arthurcgc

Reply:
?

Reply:
I didn't understand the relation of what you said with the line of code you just highlighted... Are you asking the same thing @nettoclaudio asked? If so, it doesn't break the client....

Reply:
Ok, let me explain, you changed the contract of log api renaming field appname to name, but, fortunately, tsuru-client does not care about with this field:  https://github.com/tsuru/tsuru-client/blob/master/tsuru/client/log.go#L107